app/views/inventories/index.html.erb

```diff
@@ -1 +1,23 @@
-<h1> This page would be for all the listed inventories by a given user </h1>
+<div class="container mt-3">
+  <h1 class="text-center mb-3">Your Inventories</h1>
+  <% if current_user %>
+    <%= link_to "New Inventory", new_inventory_path, class: "btn btn-primary" %>
```

Comment:
- [ ] You should be able to open add new inventory view when you click this link. At this there is no view to handle inventory creation.
- [ ] For consistency reasons, it is good to right align the button and make at the top of the list.